inginious/frontend/pages/course_admin/danger_zone.py

```diff
@@ -215,7 +216,7 @@ def get_backup_list(self, course):
 
     def page(self, course, msg="", error=False):
         """ Get all data and display the page """
-        thehash = hashlib.sha512(str(random.getrandbits(256)).encode("utf-8")).hexdigest()
+        thehash = self.user_manager.hash_password(str(random.getrandbits(256)))
```

Comment:
Is the method a class method or not ? Above you use ``UserManager.hash_password``. This is not really needed as the user manager is a singleton, but it would be nice to choose and adapt the code accordingly.